### ISSUE TITLE
Support for combined-frames GIFs

### ISSUE THREAD

**Issue by Winterhuman** [ImageOptim/gifski#308]
GIMP provides a filter option called "Optimise (for GIF)", which uses animation frames which combine/persist rather than having each frame fully replace the previous, which can significantly reduce the filesize (e.g. `1104 bytes` (after `gifski`) -> `660 bytes`, for a 9 frame pixel art animation). However, applying `gifski` to these GIFs seems to revert the optimisation.

It'd be amazing if `gifski` could retain combined-mode frames for the output, or better yet, provide a way to output combined frames natively (although it does have a limitation, at least for GIMP's filter, in that you can't replace a pixel with transparency once it's set. Unsure whether regular and combined frames can be mixed in the same GIF).

**Comment by kornelski** [comment]
gifski already uses "keep" mode to encode only differences between frames. 

1KB animation is too small to see benefits of gifski. It's geared towards full motion video that takes megabytes.

**Comment by Winterhuman** [comment]
Yep, you're right, I don't know why I didn't think to open the output in GIMP to see what mode the frames are set to (and it turns out yes, you can mix the frame types). Out of curiosity, how would I go about comparing Gifski's output and GIMP's output to figure out why GIMP's was smaller? Do you want me to send the two GIFs here?

**Comment by Winterhuman** [comment]
I've analysed the two GIFs with `gifsicle -I input.gif -o analysis.txt` and found out why GIMP's output is smaller:

**TLDR**: GIMP is saving combined frames at a much smaller resolution, and is treating colour tables differently (not sure how to parse what the output is saying exactly).

GIMP:
```
* gimp.gif 9 images
  logical screen 221x48
  global color table [16]
  background 1
  loop forever
  + image #0 221x48 transparent 7
    disposal asis delay 0.13s
  + image #1 35x5 at 90,42 transparent 7
    disposal asis delay 0.13s
  + image #2 35x5 at 90,42 transparent 7
    disposal asis delay 0.13s
  + image #3 35x5 at 90,42 transparent 7
    disposal asis delay 0.13s
  + image #4 10x5 at 90,42 transparent 7
    disposal asis delay 0.13s
  + image #5 13x5 at 92,42 transparent 7
    disposal asis delay 0.13s
  + image #6 16x5 at 94,42 transparent 7
    disposal asis delay 0.13s
  + image #7 18x5 at 97,42 transparent 7
    disposal asis delay 0.13s
  + image #8 18x5 at 102,42 transparent 7
    disposal asis delay 0.13s
```

Gifski:
```
* gifski.gif 9 images
  logical screen 221x48
  global color table [2]
  background 0
  loop forever
  + image #0 221x48 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #1 131x5 at 90,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #2 131x5 at 90,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #3 131x5 at 90,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #4 131x5 at 90,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #5 129x5 at 92,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #6 127x5 at 94,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #7 124x5 at 97,42 transparent 0
    local color table [8]
    disposal asis delay 0.13s
  + image #8 221x48 transparent 0
    local color table [8]
    disposal background delay 0.13s
```

As you said, Gifski isn't really made to handle GIFs this small, and I'm honestly not sure what to even suggest in terms of functionality for Gifski to be able to achieve these filesizes, so I'll close this issue.
